dev_tools/docs/build_api_docs.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2020 The OpenFermion Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Tool to generate external api_docs for OF (Shameless copy from TFQ)."""
+
+import os
+
+from absl import app
+from absl import flags
+from tensorflow_docs.api_generator import doc_controls
+from tensorflow_docs.api_generator import generate_lib
+from tensorflow_docs.api_generator import public_api
+
+import openfermion as of
+
+flags.DEFINE_string("output_dir", "/tmp/of_api", "Where to output the docs")
+
+flags.DEFINE_string("code_url_prefix",
+                    ("https://github.com/quantumlib/openfermion/tree/master/src"
```

Comment:
not sure it makes a difference, but `docs_to_devsite --preview` is case-sensitive for the repo name ("OpenFermion")

Reply:
Updated the link to be: https://github.com/quantumlib/OpenFermion/tree/master/src/openfermion . The url uses camel case whereas `src/openfermion` is all lower case.